resources/monitoring/templates/kyma-additions/alertmanager.config.yaml

```diff
@@ -17,19 +17,19 @@ route:
     match:
       alertname: Watchdog
   {{- if and .Values.global.alertTools.credentials.victorOps }}
-  {{- if and (not (eq "" .Values.global.alertTools.credentials.victorOps.apikey)) (not (eq "" .Values.global.alertTools.credentials.victorOps.routingkey))}}
+  {{- if and (not (eq "" .Values.global.alertTools.credentials.victorOps.apikey)) (not (eq "" .Values.global.alertTools.credentials.victorOps.routingkey)) (not (eq "" .Values.global.alertTools.credentials.victorOps.matchExpression))}}
```

Comment:
```suggestion
  {{- if and (not (eq "" .Values.global.alertTools.credentials.victorOps.apikey)) (not (eq "" .Values.global.alertTools.credentials.victorOps.routingkey))}}
```

Reply:
That condition is not needed as we have a default. So the value will be set always

Reply:
Gotcha, i update it right now.